recipes/gapseq/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set version = "1.2" %}
+
+package:
+  name: gapseq
+  version: {{ version }}
+
+source:
+  url: https://github.com/jotech/gapseq/archive/refs/tags/v{{ version}}.tar.gz
+  sha256: 061799f8e0406662450dd3d8d61d824220670fb23231895deecd490a5cbdf7cf
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('gapseq', max_pin="x") }}
+  skip: True # [osx]
+
+requirements:
+  run:
+    - r-base =4.3 # Version specification is not strictly necessary, but can help avoiding future sudden problems due to incompatible changes to R.
```

Comment:
please remove the pin, we pin those packages during the build so its not needed here.

Reply:
OK.